Source/WebKit/UIProcess/linux/MemoryPressureMonitor.cpp

```diff
@@ -438,8 +439,20 @@ size_t CGroupMemoryController::getCgroupFileValue(FILE *file)
     if (!file || fseek(file, 0, SEEK_SET))
         return notSet;
 
-    size_t value;
-    return (fscanf(file, "%zu", &value) == 1) ? value : notSet;
+    char rawValue[VALUE_BUFFER_SIZE + 1];
+    int res = fscanf(file, "%" STRINGIFY(VALUE_BUFFER_SIZE) "[^\n]", rawValue);
+
+    if (res < 1)
```

Comment:
the variable res is not really needed, we can check the return code from `fscanf` in the `if()` ifself

Reply:
valid point.  PR just updated according with this suggestion.